exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -103,6 +103,11 @@ public void Roll(int roll)
 
         }
 
+        private static bool CanLeavePenaltyBox(int roll)
+        {
```

Comment:
Good idea to add a function to make it simpler to follow. I was focused on removing code so next time I will think if I can add anything.